app/jobs/scheduled/topic_timer_enqueuer.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module Jobs
+
+  # Runs periodically to look through topic timers that are ready to execute,
+  # and enqueues their related jobs.
+  #
+  # Any leftovers will be caught in the next run, because execute_at will
+  # be < now, and topic timers that have run are deleted on completion or
+  # otherwise have their execute_at time modified.
+  class TopicTimerEnqueuer < ::Jobs::Scheduled
+    every 2.minutes
+
+    def execute(args = nil)
+      timers = TopicTimer.pending_timers
+
+      timers.each do |timer|
```

Comment:
```suggestion
      timers.find_each do |timer|
```

Just to be safer on larger sites.